src/adapters/handlers/nfts.ts

```diff
@@ -48,12 +49,16 @@ export function createNFTsHandler(
       'emoteGender',
       WearableGender
     )
-    const emotePlayMode = params.getValue<EmotePlayMode>('emotePlayMode', EmotePlayMode)
+    const emotePlayMode = params.getValue<EmotePlayMode>(
+      'emotePlayMode',
+      EmotePlayMode
+    )
     const contractAddresses = params.getAddressList('contractAddress')
     const tokenId = params.getString('tokenId')
     const itemRarities = params.getList<Rarity>('itemRarity', Rarity)
     const itemId = params.getString('itemId')
     const network = params.getValue<Network>('network', Network)
+    const status = params.getList<RentalStatus>('status', RentalStatus)
```

Comment:
Should we rename this to `rentalStatus` or something more descriptive? `status` on an `NFT` entity sounds a little bit generic. What do you think?

Reply:
You're right, I was thinking about changing it to rentalStatus, but I ended up with `status` as it was only going to be used in endpoint where the status could be filtered, but there's also the order status, so it makes sense to change it.